src/rest/QueryAnswerResponse.ts

```diff
@@ -0,0 +1,10 @@
+export interface IQueryAnswerResponse {
+  question: string;
+  answerSnippet: string;
```

Comment:
Could we rename to `answer` here? Or is `Snippet` important to mention?

Reply:
Unfortunately, this is part of the HTTP response's body, therefore I can't change it. I could encapsulate it in another interface, but this seems like it may be redundant.